tg/controllers.py

```diff
@@ -453,6 +453,22 @@ def edit_msg(self) -> None:
                     self.model.edit_message(text=text)
                     self.present_info("Message edited")
 
+    @bind(chat_handler, ["c"])
+    def view_contacts(self) -> None:
+        contacts = self.model.users.get_contacts()
+        if not contacts:
+            return self.present_error("Can't get contacts")
+
+        total = contacts["total_count"]
+        users = [f"{total} users:"]
+        for user_id in contacts["user_ids"]:
+            user = get_user_label(self.model.users, user_id)
+            status = self.model.users.get_status(user_id)
+            users.append(f"{user:<40} | {status}")
```

Comment:
40 chars could  be redundant and wasteful screen use. we can use `min(40, len(longest_contant_name))`

Reply:
Okay